openpype/modules/deadline/plugins/publish/submit_houdini_cache_deadline.py

```diff
@@ -0,0 +1,133 @@
+import os
+
+from datetime import datetime
+
+import requests
+import hou
+
+import pyblish.api
+
+from openpype.pipeline import legacy_io
+from openpype.tests.lib import is_in_tests
+from openpype.lib import is_running_from_build
+
+
+class HoudiniSubmitPublishDeadline(pyblish.api.InstancePlugin):
```

Comment:
Shouldn't this be using the abstract deadline submitter base class?

Reply:
yes, it is. And you can see the creation from the base class in #4825 but there is a overlapping betweent the ticket. This is one of the reason why it is still in draft.

Reply:
Actually, no. This has nothing to do with that particular PR since it's a different publish plug-in all together.
This PR can implement this logic on abstract deadline submitter just fine without getting in the way of the other PR.

Also, the filename being `submit_houdini_cache_deadline` and the class being `HoudiniSubmitPublishDeadline` feels a bit confusing. We should change one of them to match the other.